### PR TITLE
add italic text and project links to internship and Project sections

### PR DESCRIPTION
**Pull request by Ryadav0654**
#### References to other Issues or PRs or Relevant literature

#9 

#### Brief description of what is fixed or changed

- Added italicized text directly below the section titles to improve the visual structure and provide more context.
- Included project links in both sections, allowing users to easily access relevant projects directly from the page.

Screenshot 2024-10-01 035153

### CODE REVIEW

**Comment by Kishan-Ved** [comment]
Looks perfect! I left one minor change.

**Comment by Ryadav0654** [comment]
@Kishan-Ved ,
are you suggesting some changes ?

**Comment by Kishan-Ved** [comment]
> @Kishan-Ved , are you suggesting some changes ?

Just remove this line: `console.log("intern info:" , info);`

File: script.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ function addNewIntern() {
   // Get values from the form
   let title = document.getElementById("internTitleF").value;
   let info = document.getElementById("internInfoF").value;
+  console.log("intern info:" , info);
```
Comment (Kishan-Ved) [inline script.js]: I think we don't need this line.

Comment (Ryadav0654) [reply script.js]: @Kishan-Ved ,
i removed that line.

thanks!

**Comment by Kishan-Ved** [comment]
Amazing work! Thanks for this PR!

**Review by Kishan-Ved — APPROVED** [review]

**Comment by Kishan-Ved** [comment]
I have added the hacktoberfest and hacktoberfest-accepted labels, please check your profile on Hacktoberfest's website and let me know if you can see this PR.

Also, please star this repo, it will increase reach and we will have more contributors :)

**Comment by Ryadav0654** [comment]
@Kishan-Ved Thanks for merging! Excited to contribute more in the future.

**Pull request merged**